### PR TITLE
Adds env details to docs

### PR DESCRIPTION
**Pull request by 01painadam**
### Dataset
 - Updating env on a dataset is possible
 - It propagates that env change to all associated layers and widgets
### Layer
- It's not possible to update the env of a layer on it's own
- You can create a layer with a different env that the one of the associated dataset. If no env is provided, "production" is used
### Widget
- You can update the env of an individual widget
- You can create a widget with a different env that the one of the associated dataset. If no env is provided, "production" is used

### CODE REVIEW

File: source/includes/api/_dataset.md [hunk 1]
```diff
@@ -614,6 +614,8 @@ curl -X PATCH https://api.resourcewatch.org/v1/dataset/<dataset-id> \
     This is an authenticated endpoint!
 </aside>
 
+Note: When updating the `env` property of a dataset this way the change will cascade down to all associated layer and widget entities.
```
Comment (tiagojsag) [inline source/includes/api/_dataset.md]: is it me, or this sentence sounds weird?

File: source/includes/api/_layer.md [hunk 2]
```diff
@@ -317,6 +319,7 @@ curl -X PATCH https://api.resourcewatch.org/v1/dataset/<dataset_id>/layer/<layer
    }
 }'
 ```
+Note: it is not possible to update the `env` property of a layer this way. If you wish to alter the `env` of a layer you must update it's parent dataset - see _Update a Dataset_.
```
Comment (tiagojsag) [inline source/includes/api/_layer.md]: it's -> its

File: source/includes/api/_widget.md [hunk 3]
```diff
@@ -322,6 +322,9 @@ template     |           If it's a template            | Boolean |
 default      |       If it's default for dataset       | Boolean |                                    true - false |       No
 layerId      |   UuId of the relationship with layer   |    Text |                                   Uuid of layer |       No
 dataset      |           UuId of the dataset           |    Text |                                 Uuid of Dataset |       No
+env         |                                            Environment of the Widget. Set to 'production' by default                                               |   String |                                 Valid string |       No
+
+It is possible to create a widget that has a different `env` property to it's parent dataset.
```
Comment (tiagojsag) [inline source/includes/api/_widget.md]: it's -> its

**Review by tiagojsag — CHANGES_REQUESTED** [review]

File: source/includes/api/_dataset.md (hunk 1 above)
Comment (01painadam) [reply source/includes/api/_dataset.md]: It does, fixed!

**Pull request merged**